### DISCUSSION TITLE
v0.9.0

### DISCUSSION THREAD

**Discussion by pymq** [anywherelan/awl#112] [Announcements]
# What's Changed
* macOS support!
* ci: use separate configs for e2e tests for linux and windows by @pymq in https://github.com/anywherelan/awl/pull/100
* add .desktop file for linux by @GrigoryKrasnochub in https://github.com/anywherelan/awl/pull/96
* all: update dependencies by @pymq in https://github.com/anywherelan/awl/pull/104
* protocol: add AllowUsingAsExitNode config by @pymq in https://github.com/anywherelan/awl/pull/105
* vpn: add macos support by @pymq in https://github.com/anywherelan/awl/pull/106
* cmd/awl-tray: add macos specific root hacks by @pymq in https://github.com/anywherelan/awl/pull/107
* docs: Add script to install awl in Linux by @asmallbit in https://github.com/anywherelan/awl/pull/108
* ci: add macos test and build by @pymq in https://github.com/anywherelan/awl/pull/109
* cmd/awl-tray: add new logo icon by @pymq in https://github.com/anywherelan/awl/pull/110
* all: update libp2p to fix high severity vulnerability by @pymq in https://github.com/anywherelan/awl/pull/111

# New Contributors
* @asmallbit made their first contribution in https://github.com/anywherelan/awl/pull/108

**Full Changelog**: https://github.com/anywherelan/awl/compare/v0.8.1...v0.9.0
**Milestone**: [v0.9.0](https://github.com/anywherelan/awl/milestone/7?closed=1)

# Installation

For instructions on how to install anywherelan [see readme](https://github.com/anywherelan/awl#installation).

## Android

[awl-android-v0.9.0.apk](https://github.com/anywherelan/awl/releases/download/v0.9.0/awl-android-v0.9.0.apk)

### Windows binary builds

[awl-tray-windows-386-v0.9.0.zip](https://github.com/anywherelan/awl/releases/download/v0.9.0/awl-tray-windows-386-v0.9.0.zip)  
[awl-tray-windows-amd64-v0.9.0.zip](https://github.com/anywherelan/awl/releases/download/v0.9.0/awl-tray-windows-amd64-v0.9.0.zip)  

### macOS binary builds

[awl-tray-macos-amd64-v0.9.0.zip](https://github.com/anywherelan/awl/releases/download/v0.9.0/awl-tray-macos-amd64-v0.9.0.zip)  
[awl-tray-macos-arm64-v0.9.0.zip](https://github.com/anywherelan/awl/releases/download/v0.9.0/awl-tray-macos-arm64-v0.9.0.zip)  

### Linux binary builds

[awl-tray-linux-386-v0.9.0.tar.gz](https://github.com/anywherelan/awl/releases/download/v0.9.0/awl-tray-linux-386-v0.9.0.tar.gz)  
[awl-tray-linux-amd64-v0.9.0.tar.gz](https://github.com/anywherelan/awl/releases/download/v0.9.0/awl-tray-linux-amd64-v0.9.0.tar.gz)  
[awl-tray-linux-arm-v0.9.0.tar.gz](https://github.com/anywherelan/awl/releases/download/v0.9.0/awl-tray-linux-arm-v0.9.0.tar.gz)  
[awl-tray-linux-arm64-v0.9.0.tar.gz](https://github.com/anywherelan/awl/releases/download/v0.9.0/awl-tray-linux-arm64-v0.9.0.tar.gz)  
[awl-tray-linux-mips-v0.9.0.tar.gz](https://github.com/anywherelan/awl/releases/download/v0.9.0/awl-tray-linux-mips-v0.9.0.tar.gz)  
[awl-tray-linux-mipsle-v0.9.0.tar.gz](https://github.com/anywherelan/awl/releases/download/v0.9.0/awl-tray-linux-mipsle-v0.9.0.tar.gz)  

### Linux binary builds

[awl-linux-386-v0.9.0.tar.gz](https://github.com/anywherelan/awl/releases/download/v0.9.0/awl-linux-386-v0.9.0.tar.gz)  
[awl-linux-amd64-v0.9.0.tar.gz](https://github.com/anywherelan/awl/releases/download/v0.9.0/awl-linux-amd64-v0.9.0.tar.gz)  
[awl-linux-arm-v0.9.0.tar.gz](https://github.com/anywherelan/awl/releases/download/v0.9.0/awl-linux-arm-v0.9.0.tar.gz)  
[awl-linux-arm64-v0.9.0.tar.gz](https://github.com/anywherelan/awl/releases/download/v0.9.0/awl-linux-arm64-v0.9.0.tar.gz)  
[awl-linux-mips-v0.9.0.tar.gz](https://github.com/anywherelan/awl/releases/download/v0.9.0/awl-linux-mips-v0.9.0.tar.gz)  
[awl-linux-mipsle-v0.9.0.tar.gz](https://github.com/anywherelan/awl/releases/download/v0.9.0/awl-linux-mipsle-v0.9.0.tar.gz)  

### Windows binary builds

[awl-windows-386-v0.9.0.zip](https://github.com/anywherelan/awl/releases/download/v0.9.0/awl-windows-386-v0.9.0.zip)  
[awl-windows-amd64-v0.9.0.zip](https://github.com/anywherelan/awl/releases/download/v0.9.0/awl-windows-amd64-v0.9.0.zip)  

<hr /><em>This discussion was created from the release <a href='https://github.com/anywherelan/awl/releases/tag/v0.9.0'>v0.9.0</a>.</em>

**Comment by dillfrescott** [comment]
Awesome!!!
